util/io/subdirfs.go

```diff
@@ -0,0 +1,20 @@
+package io
+
+import (
+	"io/fs"
+	"path/filepath"
+)
+
+type subDirFs struct {
+	dir string
+	fs  fs.FS
+}
+
+func (s subDirFs) Open(name string) (fs.File, error) {
+	return s.fs.Open(filepath.Join(s.dir, name))
+}
+
+// NewSubDirFS returns file system that represents sub-directory in a wrapped file system
+func NewSubDirFS(dir string, fs fs.FS) *subDirFs {
```

Comment:
Less sure on this, but can't this be replaced with https://pkg.go.dev/io/fs#SubFS

Reply:
It does seem that you can replace this as well.

Reply:
Thanks. I've updated with your suggestion